sdds/writer.py

```diff
@@ -45,13 +46,13 @@ def _write_header(sdds_file: SddsFile, outbytes: IO[bytes]) -> List[str]:
 
 def _sdds_def_as_str(definition: Union[Description, Definition, Data]) -> str:
     start = definition.TAG + " "
-    things = ", ".join([f"{key}={definition.__dict__[key]}"
-                        for key in definition.__dict__ if "__" not in key])
+    field_values = {field.name: getattr(definition, field.name)for field in fields(definition)}
```

Comment:
forgotten space before `for` here, not sure if that makes Python cry or not

Reply:
Thanks, but python don't care